moped-editor/src/components/GridTable/FiltersChips.js

```diff
@@ -105,12 +115,34 @@ const FiltersChips = ({
       prevSearchParams.set(advancedSearchIsOrParamName, !isOr);
       return prevSearchParams;
     });
+    setIsViewSaved(false);
+  };
+
+  const handleSaveView = () => {
+    setIsSaveViewModalOpen(true);
   };
 
   return (
     <Box className={classes.filtersList}>
       <Typography className={classes.filtersText} component="span">
         <Grid container alignItems={"center"} spacing={0.5}>
+          <Grid>
+            <Tooltip
+              placement="bottom-start"
+              title="Save these filters to Dashboard"
```

Comment:
smallest preference here but my brain wants this to be a full sentence "Save these filters to the Dashboard" :thinking: